x/simulation/params.go

```diff
@@ -63,97 +33,15 @@ var (
 		{15, 92, 1},
 		{0, 3, 99},
 	})
-
-	// ModuleParamSimulator defines module parameter value simulators. All
-	// values simulated should be within valid acceptable range for the given
-	// parameter.
-	ModuleParamSimulator = map[string]func(r *rand.Rand) interface{}{
```

Comment:
this was split and moved to `x/<module>/simulation/genesis.go`, every param has now a random generator function